src/main/resources/views/CommercialVehicles/ADR_PASS.hbs

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html>
+<head>
+    <title>MOT test certificate ({{testType}})</title>
+    <style type="text/css" media="print, screen">
+            {{~> assets/stylesheets/adr this}}
+    </style>
+</head>
+<body>
+<div class="header">
+    <p>                            <span id="make">{{adrData.make}}</span>           <span id="chasisNumber">{{adrData.chasisNumber}}</span>          <span id="registrationNumber">{{adrData.registrationNumber}}</span></p>
+</div>
+<div class = "applicantDetails">
+    <h3>5 Name and business address of carrier, owner or operator</h3>
+    <p><span id="applicantDetailsName">{{adrData.applicantDetails.name}}</span> <span id="applicantDetailsStreet">{{adrData.applicantDetails.street}}</span> <span id="applicantDetailsTown">{{adrData.applicantDetails.town}}</span> <span id="applicantDetailsCity">{{adrData.applicantDetails.city}}</span> <span id="applicantDetailsPostCode">{{adrData.applicantDetails.postcode}}</span></p>
+</div>
+<div class="vehicleType">
+    <h3>6 Description of vehicle</h3>
+    <p><span id="vehicleType">{{adrData.vehicleType}}</span></p>
+</div>
+<div class="permittedDangerousGoods">
+    <h3>7 Vehicle designation(s) according to 9.1.1.2 of ADR</h3>
+    <p><span id="formattedPermittedDangerousGoods">{{adrData.formattedPermittedDangerousGoods}}</span></p>
+</div>
+<div class = "brakeEndurance">
+    <h3>8 Endurance braking system</h3>
+    {{#if adrData.brakeEndurance}}
+        <p>( )Not applicable (<img src="{{root}}/assets/images/tick.png" alt="tick" class="tick" id="tick"/>)The effectiveness according to 9.2.3.1.2 of ADR is sufficient for a total mass of transport unit of (<span id="weight">{{adrData.weight}}</span>)t</p>
+    {{else}}
+        <p>(<img src="{{root}}/assets/images/tick.png" alt="tick" class="tick" id="tick"/>)Not applicable ( )The effectiveness according to 9.2.3.1.2 of ADR is sufficient for a total mass of transport unit of (XX)t</p>
```

Comment:
Is XX supposed to be a value?

Reply:
no. xx is a value only when the 2nd option is ticked in the view(when if condition is true). when first option is ticked in the view(else branch) xx doesn't get populated.